cilantro_audit/home_page.py

```diff
@@ -7,10 +7,11 @@
 
 from cilantro_audit.admin_page import AdminPage
 from cilantro_audit.auditor_page import AuditorPage
-from cilantro_audit.completed_audits_list_page import CompletedAuditsListPage
-from cilantro_audit.constants import KIVY_REQUIRED_VERSION, ADMIN_SCREEN, HOME_SCREEN, AUDITOR_SCREEN, \
-    CREATE_AUDIT_TEMPLATE_PAGE, COMPLETED_AUDITS_LIST_PAGE
```

Comment:
Also deleted the import of this page (see other comment)

Reply:
See previous comment.